changelog/unreleased/enhancement-direct-delete

```diff
@@ -0,0 +1,6 @@
+Enhancement: Implement proper direct delete
+
+Implement a proper delete action for a single file. Previously this was being handled via selecting the file.
```

Comment:
Don't like the "previously this was being handled [...]" part too much, since it doesn't explain what's new/changed.

Reply:
Something like `We implemented a proper delete action for a single file instead of reusing the batch action for deleting multiple files`?